groundwork/plugins/gw_recipes_builder.py

```diff
@@ -52,7 +52,9 @@ def activate(self):
                                           "For more information, please take a look into the README file "
                                           "to know how to go on.\n"
                                           "For help visit: https://groundwork.readthedocs.io\n\n"
-                                          "Have fun with your groundwork package.")
+                                          "Have fun with your groundwork package.",
+                              pre_build_validation=self._pre_build_validate,
```

Comment:
should not be needed, as no real checks are performed.
So do not set it here / Do not use the parameter